docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,141 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Improve performance of highly selective `LIMIT` queries by making them finish
+  as soon as the required number of rows is produced. ({issue}`13616`)
+* Prevent using an expression which evaluates to null to `INSERT` into a
+  non-nullable column. ({issue}`13462`)
+* Reduce verbosity of query processing log by moving timeline messages to the
+  debug log level. ({issue}`13668`)
+* Fix potential query stalling on bucketed tables. ({issue}`13655`)
+* Fix potential query failure with grouped aggregations that are ordered and
+  filtered. ({issue}`13627`)
+
+## Security
+
+* Fix potential incorrect rejection of OAuth 2.0 refresh tokens. ({issue}`13168`)
+* Fix incorrectly showing outdated OAuth 2.0 refresh tokens in the web UI. ({issue}`13168`)
+
+## Docker image
+
+* Add image for `ppc64le`. ({issue}`13522`)
+
+## ClickHouse connector
+
+* Add support for the `unsupported-type-handling` catalog configuration
+  property. ({issue}`13542`)
+* Improve performance for queries with selective joins. ({issue}`13334`)
+
+## Delta Lake connector
+
+* Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Add support for the `NOT NULL` column constraint. ({issue}`13436`)
+* Fix writing incorrect results when the order of partition columns is different
+  from the order in the table definition. ({issue}`13505`)
+* Fix failure when reading a table which has partition columns renamed by 
+  another engine. ({issue}`13521`)
+
+## Druid connector
+
+* Improve performance for queries with selective joins. ({issue}`13334`)
+
+## Hive connector
+
+* Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Add support for bucket filtering on bucketed columns of `float`, `double`,
+  `date`, `list`, `map` and `bounded varchar` data types. ({issue}`13553`)
+* Add `exchange.azure.max-error-retries` configuration property for the number
+  of retries performed when accessing Azure blob storage. ({issue}`13663`)
+* Upgrade Alluxio to version 2.8.1 to address potential security
+  vulnerabilities. ({issue}`13609`)
+* Improve performance of queries with S3 Select pushdown by not utilizing
+  pushdown when it is unnecessary. ({issue}`13477`)
+* Reduce Thrift metastore communication overhead when impersonation is enabled. ({issue}`13606`)
+* Fix bug where query constraints were not used when fetching table statistics. ({issue}`13488`)
+* Fix error when writing to a table with only `date`-typed columns while using
+  the Hive metastore. ({issue}`13502`)
+* Fix error when reading a Hive view which has a column names with a reserved
+  keyword. ({issue}`13450`)
+
+## Iceberg connector
+
+* Add support for {doc}`/sql/merge`. ({issue}`7933`)
+* Add support for filtering splits based on `$file_modified_time` column
```

Comment:
"splits" is an internal concept and not relevant to release notes. It could be rephrased as:

> Improve performance when filtering on `$file_modified_time` column.